src_tests/test_uniform_losses_boson_sampling_simulator.py

```diff
@@ -55,11 +56,21 @@ def test_approximate_and_exact_distribution_distance(self) -> None:
         exact_distribution_calculator = \
             BosonSamplingWithUniformLossesExactDistributionCalculator(self.experiment_configuration)
         exact_distribution = exact_distribution_calculator.calculate_exact_distribution()
-        # TR TODO: Delta in assertion should depend on samples number!
-        approximate_distribution = self.__calculate_approximate_distribution()
+
+        number_of_samples = 1000
```

Comment:
This fragment of code looks like a duplication. Is there a chance to extract it and thus make the code more DRY?